.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"trailingComma": "none",
+	"tabWidth": 2,
+	"semi": true,
+	"singleQuote": true,
+	"useTabs": true,
```

Comment:
Can we set this to `false`? I think it'd be more standard to use spaces here.

Reply:
done!